ores/wsgi/util.py

```diff
@@ -82,3 +84,17 @@ def parse_injection(request, rev_id):
         return {rev_id: cache} if len(cache) > 0 else None
     except Exception as e:
         raise CacheParsingError(e)
+
+
+def nocache(route):
+    @wraps(route)
+    def no_cache(*args, **kwargs):
+        response = make_response(route(*args, **kwargs))
+        response.headers['Cache-Control'] = \
+            "no-store, no-cache, must-revalidate, post-check=0, " + \
+            "pre-check=0, max-age=0"
```

Comment:
The post-check and pre-check parts are IE specific and are generally frowned upon these days. And for the use case in mind possibly harmful. See  https://blogs.msdn.microsoft.com/ieinternals/2009/07/20/internet-explorers-cache-control-extensions/ where it is said

> If you want to prevent caching, do not include the post-check and pre-check directives. Doing so is completely unnecessary and results in wasted bandwidth and HTTP header processing cycles.